data-processing/entities/terms_v1/extractor.py

```diff
@@ -0,0 +1,123 @@
+from dataclasses import dataclass
+from typing import Iterator
+
+import pandas as pd
+import re
+import json
+
+import pysbd
+
+from common.parse_tex import DEFAULT_CONTEXT_SIZE, EntityExtractor, PlaintextExtractor
+from common.types import SerializableEntity
+
+
+###First convert the aggregate glossary json into a csv, this was done on a seperate file
+
+###Extracting all the terms from glossary csv
+glossary = pd.read_csv("gloss.csv")
```

Comment:
Can this be put in the `resources/` directory? https://github.com/allenai/scholar-reader/tree/master/data-processing/resources. That's where external data files are kept.